Specs/Scene/TileImagerySpec.js

```diff
@@ -0,0 +1,66 @@
+/*global defineSuite*/
```

Comment:
Oops, sorry, done.  I also opened #4413 which will prevent this from happening in the future.